demosplan/DemosPlanCoreBundle/ResourceTypes/ProcedureMapSettingResourceType.php

```diff
@@ -0,0 +1,291 @@
+            $configBuilder->territory
+                ->updatable([], function (ProcedureSettings $procedureSettings, array $territory): array {
+                    $procedureSettings->setTerritory($this->convertCoordinatesToJson($territory));
+
+                    return [];
+                })
+                ->readable(false, fn (ProcedureSettings $procedureSettings): ?array => $this->convertJsonToCoordinates($procedureSettings->getTerritory()));
+        }
+
+        if ($this->currentUser->hasPermission('field_master_procedure_default_bounding_box')) {
+            $configBuilder->defaultBoundingBox
+                ->readable(false, function (ProcedureSettings $procedureSetting): ?array {
+                    $masterTemplateMapSetting = $this->masterTemplateService->getMasterTemplate()->getSettings();
+
+                    return $this->convertFlatListToCoordinates($masterTemplateMapSetting->getBoundingBox(), 4);
+                });
+        }
+
+        $configBuilder->defaultMapExtent
+            ->readable(false, function (ProcedureSettings $procedureSetting): ?array {
+                $masterTemplateMapSetting = $this->masterTemplateService->getMasterTemplate()->getSettings();
+
+                return $this->convertFlatListToCoordinates($masterTemplateMapSetting->getMapExtent(), 4);
+            });
+
+        $configBuilder->useGlobaInformationUrl
+            ->readable(false, fn (ProcedureSettings $procedureSetting): bool => $this->globalConfig->isMapGetFeatureInfoUrlGlobal());
+
+        return $configBuilder;
+    }
+
+    protected function getSetting(string $settingName, ProcedureSettings $procedureSetting): ?Setting
+    {
+        $settings = $this->contentService->getSettings(
+            $settingName,
+            SettingsFilter::whereProcedureId($procedureSetting->getProcedure()->getId())->lock(),
+            false);
+        Assert::countBetween($settings, 0, 1);
+
+        return array_pop($settings);
+    }
+
+    protected function convertStartEndCoordinatesToFlatList(?array $coordinates): string
+    {
+        return null === $coordinates ? '' : implode(',', [
+            $coordinates['start']['latitude'],
+            $coordinates['start']['longitude'],
+            $coordinates['end']['latitude'],
+            $coordinates['end']['longitude']]);
+    }
+
+    protected function convertCoordinatesToFlatList(?array $coordinates): string
+    {
+        return null === $coordinates ? '' : implode(',', [
+            $coordinates['latitude'],
+            $coordinates['longitude']]);
+    }
+
+    protected function convertFlatListToCoordinates(string $rawCoordinateValues, $expectedCoordinatePair): ?array
+    {
+        if ('' === $rawCoordinateValues) {
+            return null;
+        }
+
+        $rawCoordinateValues = explode(',', $rawCoordinateValues);
+        $coordinateValues = [];
+
+        foreach ($rawCoordinateValues as $value) {
+            $coordinateValues[] = (float) $value;
```

Comment:
Please try `Assert::numeric();` to validate the `$value` before casting.

Reply:
8c86eef6b594e3a742c08e26d4a63e63e83247a3